static/app/views/performance/traces/content.tsx

```diff
@@ -119,6 +128,13 @@ export function Content() {
         <EnvironmentPageFilter />
         <DatePageFilter defaultPeriod="2h" />
       </PageFilterBar>
+      {hasMetric && (
+        <StyledAlert type="info" showIcon>
+          {tct('The metric query [metricQuery] is filtering the results below.', {
```

Comment:
Should we also mention the min+max value here?

Reply:
I haven't implemented the min/max part yet but will follow up on that